synapse/models/doc.py

```diff
@@ -6,35 +6,35 @@
 
             ('doc:authorable', {
```

Comment:
Do we want to capture the associated org? Thinking about a journalist who writes an op-ed for The New York Times.